app/views/languages/index.html.erb

```diff
@@ -17,12 +17,14 @@
   <% @languages.each do |language| %>
     <% works_count = estimate_number(@works_counts[language.short] || 0) %>
     <% if language == Language.default %>
-      <dt><span lang="<%= language.short %>"><%= language.name %></span></dt>
+      <dt><span lang="<%= language.short %>"><%= "#{language.name} (#{language.short})" %></span></dt>
```

Comment:
The short code will need to go outside the `span`, since the code itself is not in the language that the span is specifying. It wouldn't be a bad idea to put the short name inside an `<abbr>` tag, either.

And since we're using parentheses now, which can vary by language, this text also needs i18n. You can check out our [Internationalization Guidelines wiki page](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards) for some info on our i18n standards.